addons/website_hr_recruitment/models/hr_job.py

```diff
@@ -8,6 +8,7 @@ class Job(models.Model):
     _inherit = "hr.job"
 
     job_details = fields.Html('Process Details',
```

Comment:
Why was this added in a new field inheriting from `hr.job` while the file next to it already inherit from `hr.job` ?
See https://github.com/odoo/odoo/blame/fd355cc609cddc20e7c01a7ca12f93d6cd05c7ac/addons/website_hr_recruitment/models/hr_recruitment.py#L50
You could have moved that model inherit in a new `hr_job.py` file but having 2 inherit of the same model is to be avoided

Would also be clearer as it would be next to where the `website_description` field is declared, I was wondering why wasn't this named like that, but it already exists actually.